blue-green-spring/README.md

```diff
@@ -11,21 +11,29 @@ This sample demonstrates the following capabilities:
 * Tagging images with the current version of the artifact defined in the pom.xml file
 * Promotion of an application's container image to a blue/green production configuration
 * Switching production routes between blue and green deployments after confirmation
-* Automated rollout using the [openshift-appler](https://github.com/redhat-cop/casl-ansible/tree/master/roles/openshift-applier) Ansible role.
+* Automated rollout using the [openshift-applier](https://github.com/redhat-cop/openshift-applier/tree/master/roles/openshift-applier) Ansible role.
 
 ## Quickstart
 
 ### Requirements
-1. [OpenShift Applier](https://github.com/redhat-cop/casl-ansible)
-   `git clone git@github.com:redhat-cop/casl-ansible.git`
-   `git checkout v3.6.1`
+1. [OpenShift Applier](https://github.com/redhat-cop/openshift-applier)
```

Comment:
Would you be able to update this section to align with providing a `requirements.yml` file instead as seen [here](https://github.com/redhat-cop/container-pipelines/blob/master/multi-cluster-spring-boot/requirements.yml)?